docs/src/pages/components/data-grid/selection/selection.md

```diff
@@ -28,13 +28,14 @@ Row selection can be performed with a simple mouse click, or using the [keyboard
 ### Single row selection
 
 Single row selection is enable by default with the `DataGrid` component.
+To unselect a row, hold the <kbd class="key">CTRL</kbd> key and click on it.
 For the `XGrid`, you need to disable multiple row selection with `disableMultipleSelection={true}`.
 
 {{"demo": "pages/components/data-grid/selection/SingleRowSelectionGrid.js", "bg": "inline"}}
 
 ### Multiple row selection [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
 
-To activate multiple selection, put focus the `XGrid` component and hold the <kbd class="key">CTRL</kbd> key while selecting rows.
+To activate multiple selection, put focus on the `XGrid` component and hold the <kbd class="key">CTRL</kbd> key while selecting rows.
```

Comment:
```suggestion
To select multiple rows, focus the `XGrid` component and hold the <kbd class="key">CTRL</kbd> key while selecting rows.
```
(Cmd on Mac)

Reply:
The shift key also works for selecting multiple rows one at a time, but should instead allow range selection, for example selecting row 2, holding shift, and selecting row 5 should also select rows 2, 3 & 4.

Reply:
> The shift key also works for selecting multiple rows one at a time, but should instead allow range selection, for example selecting row 2, holding shift, and selecting row 5 should also select rows 2, 3 & 4.

Yup, we can consider this too in a separate PR.